services/index/indexmgr.go

```diff
@@ -749,6 +764,10 @@ func dropIndex(db database.DB, idxKey []byte, idxName string, interrupt <-chan s
 		if err != nil {
 			return err
 		}
+		err = dropInvalidTx(db)
+		if err != nil {
+			log.Trace(err.Error())
```

Comment:
I still not confident with the swallow of the error, and why it works for our scenario. IMO, it only hides the error.